test/extensions/filters/http/common/fuzz/uber_filter.h

```diff
@@ -60,6 +63,19 @@ class UberFilterFuzzer {
     ON_CALL(factory_context_.admin_, removeHandler(_)).WillByDefault(testing::Return(true));
   }
 
+  static void addProtoDescriptor(absl::string_view filter_name, Protobuf::Message* message) {
+    if (filter_name.find("grpc_json_transcoder") != absl::string_view::npos) {
+      envoy::extensions::filters::http::grpc_json_transcoder::v3::GrpcJsonTranscoder& config =
+          dynamic_cast<
+              envoy::extensions::filters::http::grpc_json_transcoder::v3::GrpcJsonTranscoder&>(
+              *message);
+      config.clear_services();
+      config.add_services("bookstore.Bookstore");
+      config.set_proto_descriptor(
```

Comment:
@qiwzhang Hey Wayne, very quick noob question about the GrpcJsonTranscoder filter. I was hoping you could help. This modifies configs to have valid proto_descriptors and services, but when the the filter is created, I end up with an error that the service can't be found. Is there something I need to do to make sure the `descriptor_pool_` find the service? Thank you so much!

```

[2020-04-16 19:23:50.621][14][debug][misc] [./test/extensions/filters/http/common/fuzz/uber_filter.h:139] Controlled exception transcoding_filter: Could not find 'bookstore.Bookstore' in the proto descriptor
```

Reply:
Error is thrown here (to help debug): https://github.com/envoyproxy/envoy/blob/3adf14af5ca9f34221d59d693b173d20b27c1614/source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc#L152

Reply:
Maybe the package name should be in the service name?  The error is the service name could not be found in descriptor file.

Reply:
Hmmm. `bookstore` is the package name, and `Bookstore` is the service. Will try to use `proto_descriptor_bin` instead! Thank you!